assets/_locales/en/messages.json

```
@@ -3804,6 +3804,136 @@
+    "message": "Llama Land is an AI powered MMO game built on AO.  LLama Coin is the official test token of Llama Land",
+    "description": "Description for Llama Coin token"
+  },
+  "token_description_DM3FoZUq_yebASPhgd8pEIRIzDW6muXEhxz5_JwbZwo": {
+    "message": "PIXL is a rewards token for purchasing atomic assets on the Universal Content Marketplace. When users buy atomic assets via Bazar or any UCM platform, they become eligible to receive PIXL tokens.",
+    "description": "Description for PIXL token"
+  },
+  "token_description_wOrb8b_V8QixWyXZub48Ki5B6OIDyf_p1ngoonsaRpQ": {
+    "message": "The first Meme token on AO",
+    "description": "Description for Trunk token"
+  },
+  "token_description_8rbAftv7RaPxFjFk5FGUVAVCSjGQB4JHDcb9P9wCVhQ": {
+    "message": "Transcend Keystroking, be an AGENT",
+    "description": "Description for AGENT token"
+  },
+  "token_description_agYcCFJtrMG6cqMuZfskIkFTGvUPddICmtQSBIoPdiA": {
+    "message": "ARIO (tARIO) is the native test token of the AR.IO permanent cloud network. The token incentivizes gateway operators to provide vital services that facilitate access to and utility of the permaweb. In addition, ARIO can be used for purchasing and managing Arweave Name System (ArNS) domains as well as for participation in delegated staking.",
+    "description": "Description for ARIO test token"
+  },
+  "token_description_qNvAoz0TgcH7DMg8BCVn8jF32QH5L6T29VjHxhHqqGE": {
+    "message": "ARIO is used for staking, network rewards, and purchasing decentralized smart domains within the AR.IO permanent cloud network.",
+    "description": "Description for ARIO token"
+  },
+  "token_description_aYrCboXVSl1AXL9gPFe3tfRxRf0ZmkOXH65mKT0HHZw": {
+    "message": "Level up for the AR.IO mainnet! Engage in quests with the Permanent Cloud Network to gain Experience Points (EXP), onchain tokens that represent potential eligibility for the network's forthcoming airdrops. Learn more at: https://ar.io/experience",
+    "description": "Description for EXP token"
+  },
+  "token_description_7GoQfmSOct_aUOWKM4xbKGg6DzAmOgdKwg8Kf_CbHm4": {
+    "message": "WNDR is the token of the Wander Network. WNDR is a utility token that rewards meaningful engagement across the AO and Arweave ecosystems",
+    "description": "Description for WNDR token"
+  },
+  "token_description_FBt9A5GA_KXMMSxA2DJ0xZbAq8sLLU2ak_YJe9zDvg8": {
+    "message": "Astro is the liquidity solution built for seamless integration within the AO ecosystem. By aggregating multiple stablecoins and fiat currencies into a single, unified asset - USDA, Astro ensures frictionless value exchange between real-world and on-chain applications.",
+    "description": "Description for USDA token"
+  },
+  "token_description_n2MhPK0O3yEvY2zW73sqcmWqDktJxAifJDrri4qireI": {
+    "message": "LiquidOps is a lending and borrowing protocol that providers users the ability to earn interest on their Arweave and AO assets. LQD is the governance token for the LiquidOps protocol",
```

```suggestion
    "message": "LiquidOps is a lending and borrowing protocol that provides users the ability to earn interest on their Arweave and AO assets. LQD is the governance token for the LiquidOps protocol",
```

Also check on the CN translation of this one in case this word changes the translation as well 😉

---

Addressed here https://github.com/wanderwallet/Wander/pull/966/commits/99be2cfb960b921d5e180389838a863139b8badd